test/general/macros.pass.cpp

```diff
@@ -0,0 +1,56 @@
+// -*- C++ -*-
+//===-- macros.pass.cpp --------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+
+#include <oneapi/dpl/pstl/onedpl_config.h>

```

Comment:
I would not expect the user to include it explicitly. It should be available after including either ```execution```, ```algorithm```, ```memory``` and so on. Let's remove it.

Reply:
I removed it.